modules/file-system/parallelstore/README.md

```diff
@@ -1,3 +1,80 @@
+## Description
+
+This module creates [parallelstore](https://cloud.google.com/parallelstore)
+instance. Parallelstore is Google Cloud's first party parallel file system
+service based on [Intel DAOS](https://docs.daos.io/v2.2/)
+
+### Parallelstore Quota
+
+To get access to a private preview of Parallelstore APIs, your project needs to
+be allowlisted. To set this up, please work with your account representative.
+
+### Example - New VPC
+
+For parallelstore instance, Below snippet creates new VPC and configures private-service-access
+for this newly created network.
+
+The parallelstore instance created here can be used with Slurm cluster or compute
+VM running Ubuntu 22.04, debian 12 or HPC Rocky Linux 8.
+
+```yaml
+ - id: network
+    source: modules/network/vpc
+    settings:
+      mtu: 8896
+
+  - id: ps_connect
+    source: community/modules/network/private-service-access
+    use: [network]
+    settings:
+      prefix_length: 24
+
+  - id: ps_instance
+    source: modules/file-system/parallelstore
+    use: [network, ps_connect]
+```
+
+### Example - Existing VPC
+
+If you want to use existing network with private-service-access configured, you need
+to manually provide `private_vpc_connection_peering` to the parallelstore module.
+You can get this details from pantheon in `VPC network peering` section. Below
+is the example of using existing network and creating parallelstore.
+
+```yaml
+  - id: network
+    source: modules/network/pre-existing-vpc
+    settings:
+      network_name: ${network_name} // Add network name
+      subnetwork_name: ${subnetwork_name} // Add subnetwork name
+
+  - id: ps-instance
+    source: modules/file-system/parallelstore
+    use: [network]
+    settings:
+      private_vpc_connection_peering: ${private_vpc_connection_peering}
```

Comment:
What does this name look like? Is it just string or is it uri? If uri then showing format could be helpful.

Reply:
Its basically string https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/develop/community/modules/network/private-service-access/main.tf#L39 

so you replace  private_vpc_connection_peering: "servicenetworking.googleapis.com". 
I haven't tried with configuring with some random "abcdef" string for private_vpc_connection_peering here.

Reply:
Maybe do:
```
    settings:
      private_vpc_connection_peering: <private_vpc_connection_peering> # will look like "servicenetworking.googleapis.com"
```